src/vidpf.rs

```diff
@@ -0,0 +1,735 @@
+// Copyright (c) 2024 Cloudflare, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//! Implementation of a Verifiable Incremental Distributed Point Function (VIDPF).
+//!
+//! The VIDPF construction is specified in [[draft-mouris-cfrg-mastic]] and builds
+//! on techniques from [[MST23]] and [[CP22]] to lift an IDPF to a VIDPF.
+//!
+//! [CP22]: https://doi.org/10.1007/978-3-031-06944-4_6
```

Comment:
eprint links are preferred.
```suggestion
//! [CP22]: https://eprint.iacr.org/2021/580
```